api/evidence.go

```diff
@@ -13,6 +13,8 @@ type Evidence struct {
 	ConfigID         *uuid.UUID    `json:"config_id"`
 	ConfigChangeID   *uuid.UUID    `json:"config_change_id"`
 	ConfigAnalysisID *uuid.UUID    `json:"config_analysis_id"`
+	Config           ConfigItem    `json:"config,omitempty" gorm:"foreignKey:ConfigID;references:ID"`
```

Comment:
Config and Component are optional, and I don't think we want to embed them?

Reply:
type xyz struct {
Evidence
Config
Incident
Compn
}